### PR TITLE
devtools/featuretool

### PR DESCRIPTION
**Pull request by rustyrussell**
I found this easier than reading hex.

Changelog-None

### CODE REVIEW

File: devtools/Makefile [hunk 1]
```diff
@@ -47,6 +47,8 @@ DEVTOOLS_COMMON_OBJS :=				\
 	wire/channel_type_wiregen.o		\
 	wire/tlvstream.o
 
+devtools/features: $(CCAN_OBJS) common/features.o common/utils.o wire/fromwire.o wire/towire.o devtools/features.o
```
Comment (vincenzopalazzo) [inline devtools/Makefile:50]:
```suggestion
devtools/features: $(CCAN_OBJS) common/features.o common/utils.o wire/fromwire.o wire/towire.o common/features.o
```

maybe this should work?

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
LGTM https://github.com/ElementsProject/lightning/pull/4807/commits/7d77d50d86147e3f3e97e8dcddb168a793ac49d9

Only a small comment on the Makefile.

**Comment by rustyrussell** [comment]
No, I forgot to commit the actual tool!

File: devtools/features.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/* Turns a hex string into feature. */
+#include "config.h"
+#include <ccan/err/err.h>
+#include <common/features.h>
+#include <common/utils.h>
+#include <stdio.h>
+
+int main(int argc, char *argv[])
+{
+	const u8 *features;
```
Comment (vincenzopalazzo) [inline devtools/features.c:10]:
```suggestion
	const u8 *features;
	
	setup_locale();
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
by following the GH actions suggestion I think here there is a call missing? and I'm curious to know it is used to make sure that the str operation works well.

**Comment by vincenzopalazzo** [comment]
>No, I forgot to commit the actual tool!

Ops, my mistake sorry!

**Comment by rustyrussell** [comment]
Rebase, and fix the missing setup_locale(), thanks @vincenzopalazzo

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/ElementsProject/lightning/pull/4807/commits/cd6454d9d06bcc9a21a6eeb1bcda67eabd6647b4

**Pull request merged**